sdks/java/core/build.gradle

```diff
@@ -90,4 +90,6 @@ dependencies {
   shadowTest library.java.avro_tests
   shadowTest library.java.zstd_jni
   testRuntimeOnly library.java.slf4j_jdk14
+  compile 'io.airlift:aircompressor:0.16'
```

Comment:
The other implementations that are present are either not pure java(contain jni, .c, .h files) or have licensing issues.
This solves both of these issues since it is pure java implemented and is under Apache Licence.
I am not sure if we have any other pure java implementation which is under Apache Licence.

Reply:
I asked on your original review thread if there were any alternative suggestions. LZO implementations that contain C code would be ok if they ran on the three most popular platforms (Linux, Mac, Windows).